tools/hardware/outputs/c_header.py

```diff
@@ -36,6 +36,14 @@
 #include <linker.h>         /* for BOOT_RODATA */
 #include <basic_types.h>    /* for p_region_t, kernel_frame_t (arch/types.h) */
 
+/* wrap raw physBase location constant to give it a symbolic name in C that's
```

Comment:
I think it would be helpful if this comment is much more verbose and contains some of the things you wrote in the pull request comment. Then it is more clear what's behind this without digging it up in the commit history.

Reply:
- [ ] expand comment